### ISSUE TITLE
Notification time stamp

### ISSUE THREAD

**Issue by tenebrios** [mastodon/mastodon#31254]
### Pitch

We should be able to know "when" someone likes a post, commented or followed.
When looking through notifications, I have no idea of which notifications are from today or from any other day.

### Motivation

When I turn on the phone in the morning, I should be able to scroll through the notifications and find the time stamp of where I left off the previous night.
Having the time stamp just makes looking through notifications more thorough. 
As it is, I have no idea of the last notification I saw the night before.
![Screenshot_20240801_124119_Mastodon](https://github.com/user-attachments/assets/e18f374d-0edb-4a80-86fe-717040b61434)

**Comment by renchap** [comment]
Closing, as this repository is for the built-in Mastodon web interface, and your screenshot is not from it. We are not responsible for how your client choses to display notifications.

**Comment by tenebrios** [comment]
> Closing, as this repository is for the built-in Mastodon web interface, and your screenshot is not from it. We are not responsible for how your client choses to display notifications.

Ok this brings up a few questions
1. I took a screenshot of where I made this suggestion.  As you see at the bottom, all 3 interfaces are shown. Where is it that we are to make suggestions for the android app if not there?

2. When you say "client", you are referring to server owners
![Screenshot_20240801_222940_Chrome](https://github.com/user-attachments/assets/bd77c646-8c21-4dc6-b3ee-ef33ebe2348c)
?

3. Are you saying that there are other servers that show time stamps for each notification? That is in fact something that exists somewhere on some servers?
